### PR TITLE
Updated Cameo and Chauvet fixtures to have Gobo and Gobo Rotation Preset

### PR DESCRIPTION
**Pull request by kluszon**
- Added rotation presets to color wheels and gobo wheels
- Added rotation presets to gobo rotation channels
- Added GoboMacro presets to gobo wheels
- Added ActsOn where relevant
- Added shutter and mainenance presets where previously missed

### CODE REVIEW

File: resources/fixtures/Cameo/Cameo-LED-Moving-Head-60W-CLMHRGB60W.qxf [hunk 1]
```diff
@@ -23,33 +23,33 @@
   <Capability Min="84" Max="97" Preset="ColorMacro" Res1="#6cff00">Bright green</Capability>
   <Capability Min="98" Max="111" Preset="ColorMacro" Res1="#ffd100">Amber</Capability>
   <Capability Min="112" Max="127" Preset="ColorMacro" Res1="#ff0000">Red</Capability>
-  <Capability Min="128" Max="191">Rainbow, direction of rotation forwards (speed slow - fast)</Capability>
-  <Capability Min="192" Max="255">Rainbow, direction of rotation forwards (speed slow - fast)</Capability>
+  <Capability Min="128" Max="191" Preset="RotationClockwiseSlowToFast">Rainbow, direction of rotation forwards (speed slow - fast)</Capability>
+  <Capability Min="192" Max="255" Preset="RotationClockwiseSlowToFast">Rainbow, direction of rotation forwards (speed slow - fast)</Capability>
```
Comment (mcallegari) [inline resources/fixtures/Cameo/Cameo-LED-Moving-Head-60W-CLMHRGB60W.qxf:27]: According to the manual this is backward rotation. Please fix

File: resources/fixtures/Chauvet/Chauvet-Q-Spot-260-LED.qxf [hunk 2]
```diff
@@ -24,55 +24,55 @@
   <Capability Min="102" Max="118" Preset="ColorMacro" Res1="#0000ff">Blue</Capability>
   <Capability Min="119" Max="135" Preset="ColorMacro" Res1="#7f7fff">Light blue</Capability>
   <Capability Min="136" Max="152" Preset="ColorMacro" Res1="#7fff7f">Light green</Capability>
-  <Capability Min="153" Max="255" Preset="GoboMacro" Res1="Others/rainbow.png">Rainbow or linear effect</Capability>
+  <Capability Min="153" Max="255" Preset="RotationCounterClockwise">Rainbow or linear effect</Capability>
```
Comment (mcallegari) [inline resources/fixtures/Chauvet/Chauvet-Q-Spot-260-LED.qxf:27]: Shouldn't this be clockwise?

File: resources/fixtures/Chauvet/Chauvet-Rogue-RH1-Hybrid.qxf [hunk 3]
```diff
@@ -21,8 +21,8 @@
   <Capability Min="0" Max="7" Preset="ShutterClose">Closed</Capability>
   <Capability Min="8" Max="15" Preset="ShutterOpen">Open</Capability>
   <Capability Min="16" Max="131" Preset="StrobeSlowToFast">Slow to fast</Capability>
-  <Capability Min="132" Max="167">Fast close, slow open (slow to fast)</Capability>
-  <Capability Min="168" Max="203">Slow close, fast open (slow to fast)</Capability>
+  <Capability Min="132" Max="167" Preset="RampDownSlowToFast">Fast close, slow open (slow to fast)</Capability>
```
Comment (mcallegari) [inline resources/fixtures/Chauvet/Chauvet-Rogue-RH1-Hybrid.qxf:24]: This should be RampUp

**Review by mcallegari — CHANGES_REQUESTED** [review]

**Comment by kluszon** [comment]
@mcallegari All reported problems fixed. Please review it again.

**Review by mcallegari — APPROVED** [review]

**Pull request merged**